python/pyspark/sql/connect/client.py

```diff
@@ -104,11 +224,13 @@ def fromProto(cls, pb: typing.Any) -> "AnalyzeResult":
 class RemoteSparkSession(object):
     """Conceptually the remote spark session that communicates with the server"""
 
-    def __init__(self, user_id: str, host: Optional[str] = None, port: int = 15002):
-        self._host = "localhost" if host is None else host
-        self._port = port
+    def __init__(self, user_id: str, connection_string: str = "sc://localhost"):
```

Comment:
I think we should probably have camelCase for all .. all PySpark API follows that except `functions.py` that follows SQL standard in all other languages (after the discussion)

Reply:
I'm not sure I understand, this is the constructor though? Or do you mean the argument names?

Reply:
oh yeah. I meant the argument names.

Reply:
As discussed, I will create a follow up PR that will fix the parameter names accordingly.

Reply:
Actually, maybe it's better to do it separately ..